### PR TITLE
Update Local MDM Testing Scripts and Docs

### PR DESCRIPTION
**Pull request by zacharyljones**
### Which issue this PR addresses:

[User Story 15535660: Update Scripts/Docs for Local MDM Testing](https://dev.azure.com/msazure/AzureRedHatOpenShift/_workitems/edit/15535660)

The existing MDM test scripts do not work for simply secure enabled subscriptions.

### What this PR does / why we need it:

This PR modifies the local MDM testing scripts to work in simply secure enabled subscriptions, and updates relevant documentation accordingly. It also promotes the sharing of a remote MDM enabled VM, rather than users having to create their own personal VM.

Documentation changes also correspond to #2392.

### Test plan for issue:

I used these scripts to create an MDM enabled VM in a simply secure enabled subscription and then use the VM to emit metrics to Public INT Geneva.

### Is there any documentation that needs to be updated for this PR?

This PR contains all necessary documentation updates.

### CODE REVIEW

**Comment by zacharyljones** [comment]
@dem4gus - This PR contains the script/documentation updates mentioned in #2392

**Review by s-amann — DISMISSED** [review]
lgtm, I haven't ran through them but the instructions are clear. I'll try to test them out this week.

**Review by s-amann — APPROVED** [review]

File: hack/local-monitor-testing/deploy_MDM_VM.sh [hunk 1]
```diff
@@ -2,90 +2,83 @@
...
-ssh $CLOUDUSER@$PUBLICIP "sudo firewall-cmd --reload"
+ssh -i ./secrets/mdm_id_rsa $CLOUDUSER@$MDM_VM_IP "sudo firewall-cmd --zone=public --add-port=12345/tcp --permanent"
+ssh -i ./secrets/mdm_id_rsa $CLOUDUSER@$MDM_VM_IP "sudo firewall-cmd --reload"
 
-scp $BASE/dockerStartCommand.sh $CLOUDUSER@$PUBLICIP:
-ssh $CLOUDUSER@$PUBLICIP "chmod +x dockerStartCommand.sh"
-ssh $CLOUDUSER@$PUBLICIP "sudo ./dockerStartCommand.sh &"
+scp -i ./secrets/mdm_id_rsa $BASE/dockerStartCommand.sh $CLOUDUSER@$MDM_VM_IP:
+ssh -i ./secrets/mdm_id_rsa $CLOUDUSER@$MDM_VM_IP "chmod +x dockerStartCommand.sh"
+ssh -i ./secrets/mdm_id_rsa $CLOUDUSER@$MDM_VM_IP "sudo ./dockerStartCommand.sh &"
```
Comment (jaitaiwan) [inline hack/local-monitor-testing/deploy_MDM_VM.sh:84]: Probably an out-of-scope point, but running sudo or making sudo available seems like a bad thing to be doing. I can see it was previously a part of the script before the change, so might have to be a discussion done elsewhere.

**Review by jaitaiwan — COMMENTED** [review]
LGTM, just have a query regarding what I'd consider an erroneous sudo which existed before this PR.

**Review by petrkotas — APPROVED** [review]
This is really nicely documented. I would love to see the stuff automated so we do not have to do it manually. But this is for the future. As of now LGTM and we need to test it

**Comment by hlipsig** [comment]
Very out of date. Re-open if needed.

**Pull request closed without merging**